openlane/scripts/klayout/open_design.py

```diff
@@ -70,4 +70,4 @@ def open_design(input_lefs: Tuple[str, ...], lyt: str, lyp: str, lym: str, input
 
 
 if __name__ == "__main__":
-    open_design(shlex.split(os.getenv("KLAYOUT_ARGV")))
+    open_design(shlex.split(os.environ["KLAYOUT_ARGV"]))
```

Comment:
why?

Reply:
run `shlex.split(None)` in your Python prompt for an answer

Reply:
what is the difference in return value of getenv and environ

Reply:
`getenv` returns None if it's not found, `environ` raises an IndexError